### ISSUE TITLE
Completion not working with custom PowerShell Editor Services

### ISSUE THREAD

**Issue by ivanskorikov** [ant-druha/intellij-powershell#104]
Using JetBrains Rider.

Whatever version is loaded via `File | Settings | Languages & Frameworks | PowerShell | PowerShell Editor Services`, completion only shows keywords or locally defined functions. No PS library suggestion in completion.

Once I revert to the bundled PSES v 1.10.1, completion starts suggesting PS library functions.

**Comment by ForNeVeR** [comment]
What version of PowerShellEditorServices are you trying to use? I'd like to note that so far we use _a very old one_ 1.10.1, and upgrading it to the modern one may be very tricky.

Upgrade of the embedded PSES version is in our short-term plans (#51).

Perhaps sharing the IDE log would be useful in this case (since intellij-powershell v2.2.0 we now include the PSES log in the IDE log, so if there's a terminating error, then we'll see it).

**Comment by ivanskorikov** [comment]
Blocked by #155

**Comment by ivanskorikov** [comment]
Found a workaround for #155, so I've reproduced this with the latest plugin version and PSES v. 3.11.0
[rider-logs-20231003-11550816257121191460710029.zip](https://github.com/ant-druha/intellij-powershell/files/12790100/rider-logs-20231003-11550816257121191460710029.zip)

**Comment by ForNeVeR** [comment]
I see what's wrong now: we only attach our console listeners after we connect to the process and load the session file. This means that there will be no output and even no exit code if the session could not be found.

This is, of course, wrong, and leaves us without any diagnostics in this case.

This also explains the message `PowerShell language host process exited: null`: since we were unable to connect to the process, we haven't even assigned it into a variable, and thus weren't able to extract the exit code.

Diagnostic improvements are required.

**Comment by ForNeVeR** [comment]
See my explanation in #156. I'm going to close this issue as partially fixed and partially a duplicate of #51.
